src/Drill.cs

```diff
@@ -11,6 +11,7 @@ public class Drill : Tool
 
         private readonly long _delay;
         public int WorkersStarted;
+        private readonly ChartLabels _chartLabels = new ChartLabels("LoadTestToolbox", "Elapsed Time (secs)", "Response Time (ms)");
```

Comment:
since this is already readonly, let's actually make it public and static, which would remove the need for the Get method